x-pack/test_serverless/functional/test_suites/common/data_view_mgmt.ts

```diff
@@ -14,7 +14,7 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 const archivePath = 'test/api_integration/fixtures/es_archiver/index_patterns/basic_index';
 
 export default function ({ getService, getPageObjects }: FtrProviderContext) {
-  const PageObjects = getPageObjects(['settings', 'common', 'header']);
+  const PageObjects = getPageObjects(['settings', 'common', 'header', 'home']);
```

Comment:
I think this change is not required anymore

Reply:
Good catch [803a267](https://github.com/elastic/kibana/pull/165157/commits/803a26757a5fae07c2416a288d6fbab932865a57) for changes!